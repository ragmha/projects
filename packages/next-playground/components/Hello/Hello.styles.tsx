import styled from 'styled-components'

export const StyledHello = styled.div`
  color: ${({ theme }) => theme.colors.primary};
`

export default StyledHello
