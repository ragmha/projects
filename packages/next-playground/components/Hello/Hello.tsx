import React, { FC } from 'react'
import StyledHello from './Hello.styles'
import { HelloProps } from './Hello.types'

export const Hello: FC<HelloProps> = ({ name }) => (
  <StyledHello>Hello {name}</StyledHello>
)

export default Hello
