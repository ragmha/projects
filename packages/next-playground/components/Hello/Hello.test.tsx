import Hello from './Hello'
import { render } from '@utils/test'

describe('Hello', () => {
  it('should match the snapshot', () => {
    const { container } = render(<Hello name="Miau" />)
    expect(container).toMatchSnapshot()
  })
})
