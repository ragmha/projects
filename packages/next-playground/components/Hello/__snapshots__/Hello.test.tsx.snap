// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hello should match the snapshot 1`] = `
<div>
  <div
    class="sc-AxjAm eiEaoH"
  >
    Hello 
    Miau
  </div>
</div>
`;
